index.js

```diff
@@ -8274,28 +8274,48 @@ async function main() {
 
   console.log({ eventName, sha, headSha, branch, owner, repo });
   const workflow_id = core.getInput('workflow_id', { required: true });
+  const workflow_ids = workflow_id.replace(/\s/, '').split(',');
   const token = core.getInput('access_token', { required: true });
   console.log(`Found input: ${workflow_id}`);
   console.log(`Found token: ${token ? 'yes' : 'no'}`);
   const octokit = new github.GitHub(token);
-  const { data } = await octokit.actions.listWorkflowRuns({ owner, repo, workflow_id, branch });
-  console.log(`Found ${data.total_count} runs total.`);
-  const runningWorkflows = data.workflow_runs.filter(
-    workflow => workflow.head_sha !== headSha && workflow.status !== 'completed'
-  );
-  console.log(`Found ${runningWorkflows.length} runs in progress.`);
-  for (const { id, head_sha, status } of runningWorkflows) {
-    console.log('Cancelling another run: ', { id, head_sha, status });
-    const res = await octokit.actions.cancelWorkflowRun({ owner, repo, run_id: id });
-    console.log(`Status ${res.status}`);
-  }
+
+  await workflow_ids.forEach(async (workflow_id) => {
```

Comment:
You can't await a foreach since it doesn't return anything. You'll have to do await `promise.all(workflow_ids.map())`

Reply:
I just pushed the change, is this correct?